airflow/models/dagrun.py

```diff
@@ -166,6 +164,7 @@ class DagRun(Base, LoggingMixin):
             "state",
             "dag_id",
             postgresql_where=text("state='running'"),
+            mssql_where=text("state='running'"),
```

Comment:
Airflow is dropping support for mssql, might not be worth to add anything msssql-related  https://github.com/apache/airflow/pull/34375

Reply:
I believe this got "added" because I just haven't rebased yet. I opened a draft PR for discussion & to get my changes saved remotely just in case

Reply:
We need to remember to do a rebase at some point.

Reply:
I did a rebase to get rid of the stale changes. @RNHTTR you’ll need to do a [force pull](https://stackoverflow.com/a/9589927/1376863) to fix the branch.

Reply:
shoot, missed this and force overrode your changes. I'll rebase now.